docs/xdr/features/collect/integrations/cloud_and_saas/azure/azure_ad.md

```diff
@@ -1,4 +1,3 @@
-uuid: 19cd2ed6-f90c-47f7-a46b-974354a107bb
```

Comment:
Why do you remove the id? I wonder if this deletion will not have some impact like redirection from uuid to the correct page